assets/queries/openAPI/encoding_map_key_mismatch_schema_defined_properties/query.rego

```diff
@@ -61,6 +62,7 @@ CxPolicy[result] {
 	openapi_lib.check_openapi(doc) != "undefined"
 
 	schema_properties := doc.components.responses[r].content[c].schema.properties
+	openapi_lib.content_allowed("", r)
```

Comment:
I think this does not makes sense, since components does not need response code, as you can see [here](https://swagger.io/specification/#components-object)